khmer/_khmer.cc

```diff
@@ -1584,8 +1585,11 @@ hashtable_assemble_linear_path(khmer_KHashtable_Object * me, PyObject * args)
     if (nodegraph_o) {
         stop_bf = nodegraph_o->hashbits;
     }
+    LinearAssembler assembler(hashtable);
+    KmerFilterList lst;
+    AssemblerTraverser<RIGHT> test(hashtable, start_kmer, lst);
```

Comment:
Is this left over from a previous incarnation, otherwise I think we need a comment to explain what it does.